index.html

```diff
@@ -0,0 +1,66 @@
+<html>
+  <head>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>INTENTION TIMER</h1>
+    </header>
+    <main>
+      <!-- create ALL the Left container -->
+      <section class="new-activity-section">
+        <!-- create header for left container -->
+        <h2 class="new-activity">New Activity</h2>
+        <div class="form-container">
+          <!-- create body of left container -->
+          <section class="new-act-section"></section>
+          <!-- create 3 children buttons for study, meditate, exercise inside left container -->
+        <form class="form" id="form">
+          <section class="buttons">
+            <button type="button" class="form-button study-btn" id="Study">
```

Comment:
In the buttons section, lets make sure that we delete or make kabob case the class names if we don't need to utilize both parts of the name in our CSS(i.e. before the space and after).